wayland-client/src/globals.rs

```diff
@@ -115,7 +115,7 @@ impl GlobalList {
     /// version of the returned protocol object is the lower of the maximum requested version and the advertised
     /// version.
     ///
-    /// If the lower bound of the `version` is less than the version advertised by the server, then
+    /// If the lower bound of the `version` is greater than the version advertised by the server, then
     /// [`BindError::UnsupportedVersion`] is returned.
```

Comment:
What I assume happened here is that `version` in the code is the bound version of what is found in the global registry (i.e. the version that is _available_), while "the lower bound of the `version`" is rebond to `version_start`.